src/utils/generalUtils.ts

```diff
@@ -54,4 +54,15 @@ export namespace GeneralUtils {
     export function getApiCenterWorkspacePath(): string {
         return path.join(os.homedir(), apiCenterFolder);
     }
+
+    export async function fetchDataFromLink(link: string): Promise<string> {
+        try {
+            const res = await fetch(link);
+            const rawData = await res.json();
+            return JSON.stringify(rawData);
+        }
+        catch (err) {

```

Comment:
try-catch-throw seems redundant?

Reply:
you say redundant with `registerWithTelemetry` catch logic?